src/main/java/seedu/address/model/entity/IdentificationNumber.java

```diff
@@ -17,53 +23,48 @@
     private static final String ID_PREFIX_WORKER = "W";
     private static final String ID_PREFIX_FRIDGE = "F";
 
-    private static int countOfBodies = 0;
-    private static int countOfWorkers = 0;
-    private static int countOfFridges = 0;
+    private static UniqueIdentificationNumberMaps uniqueIds = new UniqueIdentificationNumberMaps();
```

Comment:
This line in particular isn't changed, but just a thought: could line 20 (MESSAGE_CONSTRAINTS) confuse users? Currently, MESSAGE_CONSTRAINTS is shown when an invalid ID is given by the user. With the new changes, the user doesn't need to give the long-form ID anymore.